### PR TITLE
Update UoUtah

### PR DESCRIPTION
**Pull request by jay-tau**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-l.csv [hunk 1]
```diff
@@ -836,6 +836,7 @@ Luigia Aiello,Sapienza University of Rome,http://www.diag.uniroma1.it/users/luig
 Luigia Carlucci Aiello,Sapienza University of Rome,http://www.diag.uniroma1.it/users/luigia%20carlucci-aiello,NOSCHOLARPAGE
 Luis A. F. M. Ferreira,Universidade de Lisboa,http://ciisa.fmv.ulisboa.pt/en/research-labs/research-lab-10/pessoas/item/64-luis-ferreira,7eEExREAAAAJ
 Luis Antonio Brasil Kowada,UFF,http://www.professores.uff.br/kowada,8DN4mmEAAAAJ
+Luis Antonio Garcia,University of Utah,https://lagarcia.us,F6Gzg9gAAAAJ
```
Comment (jay-tau) [inline csrankings-l.csv:841]: This is being flagged as a disambiguation entry.
However, this is an alias for Luis Garcia 0001 (https://dblp.org/pid/67/136-1.html)

File: csrankings-e.csv [hunk 2]
```diff
@@ -494,6 +493,7 @@ Eric Dinet,Université Jean Monnet,https://laboratoirehubertcurien.univ-st-etien
 Eric Dubois 0002,University of Ottawa,http://www.site.uottawa.ca/~edubois,1KEbwBkAAAAJ
 Eric E. Aubanel,University of New Brunswick,http://www.cs.unb.ca/~aubanel,5KrloWkAAAAJ
 Eric E. Gilbert,University of Michigan,http://eegilbert.org,71OktocAAAAJ
+Eric Eide,University of Utah,https://users.cs.utah.edu/~eeide,b00P0C8AAAAJ
```
Comment (emeryberger) [inline csrankings-e.csv]: Research faculty, not eligible for inclusion (sorry, Eric)

File: csrankings-e.csv [hunk 3]
```diff
@@ -518,6 +519,7 @@ Eric Mercer,Brigham Young University,https://cs.byu.edu/faculty/egm,FbUhdYYAAAAJ
 Eric Mjolsness,Univ. of California - Irvine,http://emj.ics.uci.edu,tqF2ld4AAAAJ
 Eric Monteiro,NTNU,https://www.ntnu.edu/employees/eric.monteiro,T2FKQKsAAAAJ
 Eric Neufeld,University of Saskatchewan,https://www.cs.usask.ca/faculty/eric,mEQZrhcAAAAJ
+Eric Norman Eide,University of Utah,https://users.cs.utah.edu/~eeide,b00P0C8AAAAJ
```
Comment (emeryberger) [inline csrankings-e.csv]: See above

File: csrankings-f.csv [hunk 4]
```diff
@@ -272,6 +272,8 @@ Feng Zhu 0006,UESTC,http://faculty.uestc.edu.cn/williamzhu/en/jsxx/262241/jsxx/j
 Feng-Hao Liu,Washington State University,https://school.eecs.wsu.edu/faculty/profile/?nid=feng-hao.liu,vvWcl-wAAAAJ
 Feng-Jian Wang,National Chiao Tung University,https://www.cs.nctu.edu.tw/members/detail/fjwang,NOSCHOLARPAGE
 Fengbo Ren,Arizona State University,https://search.asu.edu/profile/2436287,f-wp99AAAAAJ
+Fengguang Song,IUPUI,http://cs.iupui.edu/~fgsong,BoVnm1wAAAAJ
```
Comment (emeryberger) [inline csrankings-f.csv]: Please remove this entry

File: csrankings-r.csv [hunk 5]
```diff
@@ -966,6 +966,7 @@ Robert Powell,Durham University,https://www.dur.ac.uk/computer.science/staff/pro
 Robert R. Kessler,University of Utah,https://www.cs.utah.edu/~kessler,j9XJD8QAAAAJ
 Robert Rand 0001,University of Chicago,http://rand.cs.uchicago.edu,jYrnZVwAAAAJ
 Robert Reed,Vanderbilt University,http://engineering.vanderbilt.edu/bio/robert-reed,oEQcOy8AAAAJ
+Robert Ricci,University of Utah,https://ricci.io,zHHD40YAAAAJ
```
Comment (emeryberger) [inline csrankings-r.csv]: Research faculty, not eligible for inclusion. Please remove

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-e.csv [hunk 6]
```diff
@@ -505,6 +504,7 @@ Eric G. Mercer,Brigham Young University,https://cs.byu.edu/faculty/egm,FbUhdYYAA
 Eric Gilbert,University of Michigan,http://eegilbert.org,71OktocAAAAJ
 Eric Granger,ETS Montreal,https://etsmtl.ca/Professeurs/egranger/Accueil?lang=en-CA,TmfbdagAAAAJ
 Eric Harley,Toronto Metropolitan University,https://www.scs.torontomu.ca/~eharley,NOSCHOLARPAGE
+Eric Heisler,University of Utah,https://ericheisler.github.io,FUxJhrIAAAAJ
```
Comment (emeryberger) [inline csrankings-e.csv]: Not TT faculty; remove.

File: csrankings-d.csv [hunk 7]
```diff
@@ -561,6 +562,7 @@ David E. Breen,Drexel University,https://www.cs.drexel.edu/~david,d5PqNPgAAAAJ
 David E. Carlson,Duke University,http://carlson.pratt.duke.edu,R9FqK9cAAAAJ
 David E. Evans 0001,University of Virginia,http://www.cs.virginia.edu/~evans,gzdrT_4AAAAJ
 David E. Irwin 0001,University of Massachusetts Amherst,http://www.ecs.umass.edu/~irwin,NkwzF2wAAAAJ
+David E. Johnson 0001,University of Utah,https://users.cs.utah.edu/~dejohnso/index.shtml,HZv4Np8AAAAJ
```
Comment (emeryberger) [inline csrankings-d.csv]: Not TT faculty; remove.

File: csrankings-b.csv [hunk 8]
```diff
@@ -212,6 +212,7 @@ Ben H. H. Juurlink,TU Berlin,https://www.aes.tu-berlin.de/menue/mitarbeiterinnen
 Ben Hardekopf,Univ. of California - Santa Barbara,http://www.cs.ucsb.edu/~benh,NOSCHOLARPAGE
 Ben He,Chinese Academy of Sciences,http://www.icip.org.cn/team/heben,EmTFUdQAAAAJ
 Ben Hermann 0001,TU Dortmund,https://sse.cs.tu-dortmund.de/team/ben-hermann/,r5ZMp3MAAAAJ
+Ben Jones,University of Utah,https://users.cs.utah.edu/~benjones,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-b.csv]: Not TT faculty; remove.

**Review by emeryberger — CHANGES_REQUESTED** [review]
All faculty given here who have "Lecturer" by their name are not TT faculty and need to be removed. https://www.cs.utah.edu/people/faculty/

**Comment by jay-tau** [comment]
Fixed @emeryberger 
Is there any resource that lists the various types of non-tenure type faculty? Different universities seem to follow different naming conventions, making it rather confusing :/

**Comment by emeryberger** [comment]
"Lecturer" in US institutions is a strong indicator of non-TT (as is "Research" by a title, anywhere).

**Pull request merged**